src/index.html

```diff
@@ -6,7 +6,362 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <!-- #region header -->
+    <header class="header">
+      <div class="logo">
+        <a href="#home">
+          <img
+            src="images/logo.png"
+            alt="logo"
+            class="logo__image"
+          >
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- #endregion -->
+    <!-- #region main -->
+    <main class="page__main main">
+      <div class="card" data-qa="card">
+        <img class="card__image"
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 &quot; Retina 5K Monoblock (MNED2UA/A)">
+        <h2 class="card__name">APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)</h2>
+        <div class="card__code">Product code: 195434</div>
+        <div class="card__reviews">
+          <div class="stars-line">
+            <div class="stars-line__star"></div>
+            <div class="stars-line__star"></div>
+            <div class="stars-line__star"></div>
+            <div class="stars-line__star"></div>
+            <div class="stars-line__star"></div>
+          </div>
+          <span class="count">
+            Reviews: 5
+          </span>
+        </div>
+        <div class="card__price">
+          <div class="price">
```

Comment:
Do you need extra `price` container? Also, don't create extra BEM blocks, you don't need `price` as separate BEM block, it's just the part of the card